aspnetcore/mvc/views/tag-helpers/builtin/resources/ImageTagHelper.md

```diff
@@ -0,0 +1,46 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
```

Comment:
@GuardRex not following what to do.  Leaving as is. Maybe some fancy link syntax?